pkg/utils/utils.go

```diff
@@ -161,3 +161,43 @@ func GenerateControllerID(typeID uint16) uint16 {
 		return ctrlID
 	}
 }
+
+func CtLabelDecode(label []byte) (uint64, uint64, uint64) {
+	// Bit Order Example:
+	//
+	// No.1 1010 0010 1111 0001 0xA2F1  -  ovs register order
+	// No.2 1000 1111 0100 0101 0x8F45  -  ovs dpctl/dump-conntrack (left-right mirror from No.1)
+	// No.3 0100 0101 1000 1111 0x458F  -  netlink ct label
+	//
+	// label retrieve from netlink ct label, transfer it with little endian
+	// In the above case, it seems as No.2
+	// Since binary lib could only handle uint64, label (128 bits) split into TWO parts.
+	//
+	// The round number stores in high 10 bits. Here it means the right 10 bits in uint64 partA.
```

Comment:
// The round number stores in high 4 bits. Here it means the right 4 bits in uint64 partA.
modify 10 bits to 4 bits, forgot change comment last time (￣.￣)